avaframe/com1DFA/com1DFA.py

```diff
@@ -625,11 +639,31 @@ def initializeSimulation(cfg, demOri, inputSimLines, logName, relThField):
     demOri, dem = initializeMesh(cfgGen, demOri, methodMeshNormal)
 
     # ------------------------
+    # Initialize main release area
     log.debug('Initializing main release area')
     # process release info to get it as a raster
-    releaseLine = inputSimLines['releaseLine']
-    # check if release features overlap between features
-    prepareArea(releaseLine, demOri, thresholdPointInPoly, combine=True, checkOverlap=True)
+    if cfg['GENERAL'].getboolean('iniStep'):
+        releaseLine = inputSimLines['releaseLineBuffer']
+        releaseLineReal = inputSimLines['releaseLine']
+        # check if release features overlap between features
+        prepareArea(releaseLineReal, demOri, thresholdPointInPoly, combine=True, checkOverlap=True)
+        buffer1 = cfg['GENERAL'].getfloat('sphKernelRadius') * 0.275 * cfg['GENERAL'].getfloat('bufferZoneFactor')
+        if len(relThField) == 0:
+            # if no release thickness field or function - set release according to shapefile or ini file
+            # this is a list of release rasters that we want to combine
+            releaseLineReal = prepareArea(releaseLineReal, demOri, buffer1, thList=releaseLineReal['thickness'],
+                combine=True, checkOverlap=False)
+        else:
+            # if relTh provided - set release thickness with field or function
+            releaseLineReal = prepareArea(releaseLineReal, demOri, buffer1, combine=True, checkOverlap=False)
+            releaseLineReal['rasterData'] = releaseLineReal['rasterData'] * relThField
+
+    else:
```

Comment:
maybe we could make a small function out of this, the initializeSimulation is already pretty long